websites/N/Ngewibu/presence.ts

```diff
@@ -0,0 +1,90 @@
+const presence = new Presence({
+		clientId: "1104875968134397963",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/UMRqZDd.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ href, pathname, search } = document.location,
+		video = document.querySelector<HTMLVideoElement>("video");
+	if (pathname.startsWith("/search")) {
+		presenceData.details = "Searching for";
+		presenceData.state = search.split("?q=")[1];
+		presenceData.smallImageKey = Assets.Search;
+	} else if (pathname.startsWith("/play")) {
+		const thumbnail = document
+				.querySelector<HTMLMetaElement>('meta[property="og:image"]')
+				.getAttribute("content")
+				.replace("webp", "png"),
+			title = document
+				.querySelector<HTMLMetaElement>('meta[property="og:title"]')
+				.getAttribute("content")
+				.replace(" - NGEWIBU.TV", "");
+
+		presenceData.largeImageKey = thumbnail || presenceData.largeImageKey;
+		presenceData.state = `Episode ${title
+			.match(/E \d+/g)[0]
+			.replace("E ", "")}`;
+		presenceData.details = `Watching ${title}`;
+		if (video) {
+			presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
+			presenceData.smallImageText = video.paused ? "Paused" : "Playing";
+			presenceData.buttons = [{ label: "Watch Video", url: href }];
+		} else presenceData.buttons = [{ label: "View Anime", url: href }];
+	} else if (pathname.startsWith("/video/")) {
+		const thumbnail = document
+			.querySelector<HTMLMetaElement>('meta[property="og:image"]')
+			.getAttribute("content")
+			.replace("webp", "png");
+
+		presenceData.largeImageKey = thumbnail || presenceData.largeImageKey;
+		presenceData.state = "Shorts Video";
+		presenceData.details = `Watching ${document
+			.querySelector<HTMLMetaElement>('meta[property="og:title"]')
+			.getAttribute("content")
+			.replace(" - NGEWIBU.TV", "")}`;
+		if (video) {
+			presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
+			presenceData.smallImageText = video.paused ? "Paused" : "Playing";
+			presenceData.buttons = [{ label: "Watch Video", url: href }];
+		} else presenceData.buttons = [{ label: "View Videos", url: href }];
+	} else if (pathname === "/") {
+		presenceData.details = "Viewing home page";
+		presenceData.smallImageKey = Assets.Viewing;
+		presenceData.buttons = [{ label: "Viewing home page", url: href }];
```

Comment:
[Buttons](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) that lead to home/main pages are not allowed.
```suggestion
```

Reply:
updated